manuscript.Rmd

```diff
@@ -76,95 +90,99 @@ We would like to respond with a quote from the Science Code Manifesto (http://sc
+To assess this, we propose to simply provide a checkbox, in which the author indicates whether a scientific software package is actively maintained (e.g., software has a regular release cycle or an update within the last six months).
+In addition, authors have a chance to explain why their active maintenance may be different from these guidelines.
 
+## ID 9: Citations
+
+@gartner2022responsible suggest the number of citations to evaluate the impact of a software package.
+However, citations suffer from serious shortcomings when used to evaluate the impact of research software, particularly, when used comparatively in the context of hire and promotion.
+For example, it is much more commonly accepted to cite data analysis packages than supporting packages (such as papaja [@R-papaja] used to render this article). 
+Further, functionality of successful, modular scientific software is ideally reused in other software packages to avoid duplication of code and enable faster development of new software.
+While we highly encourage this from both a software engineering perspective and for scientific progress, it challenges the validity of citation-based metrics for impact.
+For example, consider the NLopt optimization suite, which counts 1,711 citations on Google scholar at the time of writing.
+NLopt is a backbone for many scientific model fitting packages both because it implements various optimization algorithm but also because it is open-source and can inspire re-implementations of these algorithms.
+For example, consider the famous lme4 package [@bates2014fitting] for generalized linear mixed-effects models, that is partly based on NLopt.
+This package has more than 58,000 citations on Google scholar, yet it is unlikely that researchers will cite the underlying optimization algorithm.
+<!--For example, the pdc package [@pdc-paper] offers functions to cluster time-series based on relative complexity.
+As another example, the TSclust package [@TSclust] is a wrapper to several dissimilarity measures for clustering time series. 
+Some of us maintain the pdc clustering package [@pdc-paper], which is contained in TSclust and we saw that researchers use the pdc approach but only cite the TSclust package.
+-->
 ## ID 11: GitHub Stars
 
 We object to the use of Github stars as a general metric.
-First of all, it is tied only to a single, commercial platform and allows no freedom to choose free alternatives such as local Gitlab instances or other platforms.
-More importantly, we question the validity of such metrics that like rating systems on commercial platforms are too easy to be gamed by creating fake accounts that provide high ratings.
-
-## ID 6: Date of most recent substantive update
-<!-- see issue #7 -->
+First of all, it is tied only to a single, commercial platform.
+More importantly, we question the validity of such metrics that, like rating systems on commercial platforms, are too easy to be gamed.
```

Comment:
I don't agree. Gaming the system is always easy. I'd rather criticize that software has very different ways of making an impact, therefore the merit statement. A backend library might not bost with many stars but many dependencies. Stars may be one way of showing the impact but can't be generally accepted.

Reply:
do you recommend to entirely drop the criticism of the stars?

Reply:
If we still have space issues yes, I'd move it into l. 226 together with download numbers, etc.

Reply:
good idea!